admin/server/github.go

```diff
@@ -853,9 +1000,298 @@ func (s *Server) fetchReposForInstallation(ctx context.Context, client *github.C
 	return repos, nil
 }
 
+func (s *Server) pushToGit(ctx context.Context, copyData func(projPath string) error, repo, branch, subpath, token string, force bool) error {
+	ctx, cancel := context.WithTimeout(ctx, archivePullTimeout)
+	defer cancel()
+
+	// generate a temp dir to extract the archive
+	dir, err := os.MkdirTemp(os.TempDir(), "dest_git_repos")
```

Comment:
Given this is used for both upload and git flow so we can just change the name to `project` or similar?